plugin/pkg/auth/authorizer/node/node_authorizer.go

```diff
@@ -138,8 +142,15 @@ func (r *NodeAuthorizer) Authorize(ctx context.Context, attrs authorizer.Attribu
 			return r.authorizeCSINode(nodeName, attrs)
 		case resourceSlice:
 			return r.authorizeResourceSlice(nodeName, attrs)
+		case nodeResource:
+			if r.features.Enabled(features.AuthorizeNodeWithSelectors) {
```

Comment:
It requires AuthorizeWithSelectors to be enabled. Should we also check this feature-gate in this line?

Reply:
validation at startup ensures AuthorizeNodeWithSelectors cannot be enabled if AuthorizeWithSelectors is disabled